packages/slice-machine/tests/server/getAndSetUserProfile.test.ts

```diff
@@ -15,7 +15,7 @@ jest.mock(`fs`, () => {
 describe("setShortId", () => {
   afterEach(() => {
     vol.reset();
-    fetchMock.reset();
+    nock.cleanAll();
```

Comment:
nock.restore ?
https://github.com/nock/nock#memory-issues-with-jest

Reply:
I believe I tried restore once and it failed telling me on the next nock that the URL was already define.